apps/cms-v2/src/modules/providers/contentful/contentful-config-form.tsx

```diff
@@ -29,29 +28,57 @@ const PureForm = ({
   onSubmit(values: FormSchema): void;
   onDelete?(): void;
 }) => {
-  const {
-    control,
-    getValues,
-    setValue,
-    watch,
-    handleSubmit,
-    formState: { errors },
-  } = useForm<FormSchema>({
-    defaultValues: defaultValues,
-    resolver: zodResolver(ContentfulProviderConfig.Schema.Input.omit({ type: true })),
-  });
+  const { notifyError } = useDashboardNotification();
+
+  const { control, getValues, setValue, watch, handleSubmit, setError, clearErrors } =
+    useForm<FormSchema>({
```

Comment:
**suggestion (non-blocking)**: I think you can skip the type casting if you are providing both `defaultValues` and `resolver`.